### PR TITLE
Handling conversion of date with time zone to epoch time.

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23104

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Bargenish** [comment]
Before:
Screen Shot 2020-03-30 at 15 17 24

After:
Screen Shot 2020-03-30 at 15 16 59

File: Scripts/FormattedDateToEpoch/FormattedDateToEpoch.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-date_value = demisto.args()['value']
-formatter = demisto.args()['formatter']
+epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
+
+args = demisto.args()
+date_value = args['value']
+formatter = args['formatter']
 
 date_obj = datetime.strptime(date_value, formatter)
+unix_time = int(date_obj.strftime('%s') if date_obj.tzinfo is None else (date_obj - epoch).total_seconds())
```
Comment (yaakovi) [inline Scripts/FormattedDateToEpoch/FormattedDateToEpoch.py]: move to a function `date_to_epoch(date, formatter)`
add unit tests

Comment (Bargenish) [reply Scripts/FormattedDateToEpoch/FormattedDateToEpoch.py]: Done

**Comment by kirbles19** [comment]
@Bargenish Done.

**Pull request merged**